numpy/f2py/tests/test_array_from_pyobj.py

```diff
@@ -30,20 +35,17 @@ def setup_module():
     if not util.has_c_compiler():
         pytest.skip("No C compiler available")
 
+    if not util.has_f90_compiler():
+        pytest.skip("No Fortran 90 compiler available")
+
+    if sys.platform == 'cygwin':
+        pytest.skip("Meson too old for cygwin CI")
```

Comment:
Is this still true after the release of meson 1.3?

Reply:
No but it isn't needed...